numpy/_core/_internal.py

```diff
@@ -860,6 +861,9 @@ def _prod(a):
 
 def _gcd(a, b):
     """Calculate the greatest common divisor of a and b"""
+    if not (math.isfinite(a) and math.isfinite(b)):
+        raise ValueError('Can only find greatest common divisor of '
+                            f'finite arguments, found "{a}" and "{b}"')
```

Comment:
```suggestion
        raise ValueError('Can only find greatest common divisor of '
                         f'finite arguments, found "{a}" and "{b}"')
```

Alignment is off here